examples/notebooks/validation/neuromodulation/neuromodulation_sim.py

```diff
@@ -7,7 +7,7 @@
 def analyse(network_path):
     
     
```

Comment:
it is not clear why this update is done and what it is doing

Reply:
i.e. why iSPN is removed from the validation

Reply:
The validation directory is the validations against experimental data, pipeline is being set up by Shailesh